apps/prairielearn/elements/pl-file-preview/pl-file-preview.mustache

```diff
@@ -22,17 +22,17 @@
               <span class="mr-2 js-toggle-show-preview-text">Show preview</span>
               <i class="file-preview-icon fa fa-angle-down"></i>
             </button>
-            <button type="button" class="btn btn-outline-secondary btn-sm file-preview-expand">
-              <span class="mr-2 js-toggle-expand-preview-text">Expand</span>
-              <i class="fa-solid fa-expand"></i>
-              <i class="fa-solid fa-compress d-none"></i>
-            </button>
           </div>
         </div>
         <div class="alert alert-danger mt-2 d-none error" role="alert"></div>
         <div class="file-preview collapse" id="file-preview-contents-{{uuid}}-{{index}}">
           <img class="mw-100 mt-2 d-none">
           <pre class="bg-dark text-white rounded p-3 mt-2 mb-0 d-none"><code></code></pre>
+          <button type="button" class="btn btn-outline-secondary btn-sm btn-block d-none file-preview-expand">
+            <span class="mr-2 js-toggle-expand-preview-text">Expand</span>
+            <i class="fa-solid fa-expand"></i>
+            <i class="fa-solid fa-compress d-none"></i>
```

Comment:
We usually put the icon before the text. I assume this came after for consistency with Show Preview, but since this is no longer next to it should it be moved back to before the text?

(Nit, of course, won't block on this)

Reply:
I moved the icon before the text in 1ff2248.